### DISCUSSION TITLE
Docker (Portainer) | Error 1045 | Access denied for user 'photoprism'@'172.18.0.3' (using password: YES)

### DISCUSSION THREAD

**Discussion by Zutyosh** [photoprism/photoprism#3834] [Q&A]
Hello, today I tried to deploy PhotoPrism with Portainer following the [official tutorial](https://docs.photoprism.app/getting-started/portainer/), but after completing the deployment I couldn't access the service, so I went to check the logs and came across this message:
/opt/photoprism/bin/photoprism start
time="2023-10-21T17:10:57Z" level=error msg="Error 1045: Access denied for user 'photoprism'@'172.18.0.3' (using password: YES)"

I tried to search by myself is to find this [old post](https://github.com/photoprism/photoprism/issues/67) about the same error, but I couldn't really determine the solution.

If anyone could help me, I'd be very grateful!

**My stack.yml :** 

version: '3.5'

services:
  photoprism:
    image: photoprism/photoprism:latest
    stop_grace_period: 10s
    depends_on:
      - mariadb
    security_opt:
      - seccomp:unconfined
      - apparmor:unconfined
    ports:
      - "2342:2342"
    env_file:
      - stack.env
    working_dir: "/photoprism"
    volumes:
      - "/mnt/ssd/apps_docker/photoprism/originals:/photoprism/originals"
      - "/mnt/ssd/apps_docker/photoprism/storage:/photoprism/storage"

  mariadb:
    image: mariadb:10.11
    restart: unless-stopped
    stop_grace_period: 5s
    security_opt:
      - seccomp:unconfined
      - apparmor:unconfined
    command: mariadbd --innodb-buffer-pool-size=512M --transaction-isolation=READ-COMMITTED --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci --max-connections=512 --innodb-rollback-on-timeout=OFF --innodb-lock-wait-timeout=120
    volumes:
      - "/mnt/ssd/apps_docker/photoprism_database:/var/lib/mysql"
    env_file:
      - stack.env

![image](https://github.com/photoprism/photoprism/assets/69845574/e3a06d45-4fc0-4786-a1c0-b424807ac38e)

**Comment by lastzero** [comment]
https://docs.photoprism.app/getting-started/troubleshooting/mariadb/#cannot-connect

**Reply by Zutyosh** [reply]
I've tried several of the proposed solutions, but I can't seem to solve the problem. I'm even having trouble identifying what's wrong.

**Reply by dino-zarafonitis** [reply]
This answer solved my issue https://github.com/photoprism/photoprism/issues/67#issuecomment-752894229
